### ISSUE TITLE
Implement a HTTP of-watchdog variant

### ISSUE THREAD

**Issue by alexellis** [booyaa/openfaas-rust-template#3]
Hi after the initial success of this template I want to suggest that you add another one to this repo with a suffix of http. The new template would use the new of-watchdog from OpenFaaS which allows for much high throughput by not forking a process for each request.

For an example see the openfaas-incubator org and the Go HTTP template. 

Would you like to work on this or should I try to find another Rust developer to cover it? Thanks :)

Alex

**Comment by booyaa** [comment]
Yup will give it a go (want to keep my hand in with Rust and OpenFaaS) ;)

**Comment by alexellis** [comment]
Great to hear back from you :-) This is how I structured it for Golang: https://github.com/openfaas-incubator/golang-http-template/tree/master/template/golang-http

Btw your template is part of the official template store now, available via `faas-cli template store list`

I'd love to send you some stickers and t-shirt if I have any left, email me: alex@openfaas.com 

Alex

**Comment by pablogrs** [comment]
@booyaa I wouldn't mind to contribute to that idea, would it make sense to make me a contributor?
